### DISCUSSION TITLE
Gold modules

### DISCUSSION THREAD

**Discussion by Thirteen0130** [PavelZinchenko/event-horizon-frontier-builds#55] [General]
What will happen to the gold modules (-50% power consumption, damage, range)? What fate awaits them? Will they stay or will they be removed? If removed, the balance of the game will change

Что будет с золотыми модулями (-50% энергопотребления, урон, дальность)? Какая судьба их ждёт? Они останутся или их уберут? Если уберут, то баланс игры изменится

**Comment by PavelZinchenko** [comment]
I won't be making any changes to them in this update
